will-kitchell/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "gulppackage",
+  "version": "1.0.0",
+  "description": "adding a gulpfile and a package.json",
+  "main": "index.js",
+  "directories": {
+    "test": "test"
+  },
+  "dependencies": {
+    "chai": "^3.4.0"
+  },
+  "devDependencies": {
+    "gulp": "^3.9.0",
+    "gulp-jshint": "^1.11.2"
+  },
```

Comment:
It looks like you're missing some devDependencies, such as jshint, mocha, and gulp-mocha.